packages/fast-components-class-name-contracts-msft/src/context-menu-item/index.ts

```diff
@@ -0,0 +1,12 @@
+import { ContextMenuItemClassNameContract as BaseContextMenuItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+/**
+ * The class name contract for the button component
```

Comment:
typo button -> context menu item

Reply:
yeeeesh I missed a lot of these - thanks @nalogiudice!